.github/workflows/build.yaml

```diff
@@ -42,17 +41,17 @@ jobs:
         uses: azure/setup-helm@v3.3
 
       - name: Fetch dependencies from cache
-        id: cache-yarn
+        id: cache-root-node-mods
         uses: actions/cache@v3
         with:
-          path: "**/node_modules"
-          key: yarn-build-${{ hashFiles('**/yarn.lock') }}
+          path: "node_modules"
+          key: yarn-build-${{ hashFiles('yarn.lock') }}
           restore-keys: |
             yarn-build-
 
       - name: Install dependencies
-        if: steps.cache-yarn.outputs.cache-hit != 'true'
-        run: yarn --frozen-lockfile
+        if: steps.cache-root-node-mods.outputs.cache-hit != 'true'
+        run: SKIP_SUBMODULE_DEPS=1 yarn --frozen-lockfile
```

Comment:
🧹 I realized we don't need the submodules checked out to run `fmt` or `lint` so I think we can skip installing submodule deps here.